packages/fxa-auth-server/lib/senders/emails/auth.ftl

```diff
@@ -12,11 +12,9 @@
 -brand-firefox = Firefox
 
 ## "Accounts" can be localized, "Firefox" must be treated as a brand.
--product-firefox-accounts = Firefox Accounts
+-product-firefox-accounts = Firefox accounts
 
 ## "Account" can be localized, "Firefox" must be treated as a brand.
-## This is used to refer to a user's account, e.g. "update your Firefox account ..."
-## NOTE: We currently sometimes we refer to  "Firefox Account" rather than "Firefox account". This is a known issue (FXA-4071 / fxa/issues/10654).
 -product-firefox-account = Firefox account
```

Comment:
Per [this comment](https://github.com/mozilla/fxa-content-server-l10n/pull/527#pullrequestreview-781331754), we shouldn't do something like `{ -brand-firefox } account`, so I left these as `-product-firefox-accounts` and `-product-firefox-account` with a comment above them similar to what we have in settings.